docs/source-2.0/tutorials/full-stack-tutorial.rst

```diff
@@ -0,0 +1,880 @@
+
+    ...
+    service CoffeeShop {
+        ...
+        resources: [
+            Order
+        ]
+    }
+
+Finally, you might be wondering why we did not model our coffee or menu as a resource. For our service, we are not
+exposing any functionality related to the *lifecycle* of these entities. However, let's describe a hypothetical
+example. We decide a coffee has properties like origin, roast, and tasting notes. Also, we choose to expose operations
+for adding, updating, and removing coffees. In this case, coffee would be a prime candidate for modeling as a resource.
+
+Building the model
+==================
+The model for our coffee service is complete. Before we build the model, let's take a moment and learn how to configure
+the build. The :ref:`smithy-build.json configuration file <smithy-build-json>` is how we instruct Smithy to build the
+model. A :ref:`projection <projections>` is a version of a model based on a set of :ref:`transformations <transforms>`.
+Plugins can be applied to a projection to produce artifacts based on its "version" of the model.
+For our model, we will not configure any explicit projections, since Smithy always builds the ``source`` projection.
+The ``source`` projection does not have any transformations applied, and its output includes the artifacts of
+plugins applied at the root. To build the model, run:
+
+.. code-block:: sh
+    :caption: ``/bin/sh``
+
+    smithy build
+
+Building the model will render artifacts under the ``build/smithy`` directory. Under it, The ``source`` directory
+corresponds to the output (or "build artifacts") of the ``source`` projection. With the current configuration, Smithy
+will produce the model in its :ref:`JSON AST representation <json-ast>`, and a ``sources`` directory which contains the
+model files used in the build. Additional artifacts are produced by configuring plugins, and
+:doc:`code-generators <../guides/using-code-generation/index>` are prime examples of this.
+
+Generating the server SDK
+=========================
+The server SDK is a code-generated component which provides built-in serialization, request-handling, and
+scaffolding (or "stubs") for our service. It facilitates the implementation of the service by
+providing these things, and allowing the implementer to focus on the business logic. Let's generate the server SDK
+for our service by using the following build configuration:
+
+.. code-block:: json
+    :caption: ``smithy-build.json``
+
+    {
+        "version": "1.0",
+        "sources": ["model/"],
+        "maven": {
+            "dependencies": [
+                "software.amazon.smithy:smithy-aws-traits:1.50.0",
+                "software.amazon.smithy:smithy-validation-model:1.50.0",
+                "software.amazon.smithy.typescript:smithy-aws-typescript-codegen:0.22.0"
+            ]
+        },
+        "plugins": {
+            "typescript-ssdk-codegen": {
+                "package" : "@com.example/coffee-service-server",
```

Comment:
Just an idea:
```suggestion
                "package" : "@com.example/coffee-shop-server",
```
and same for the code block when configuring the client generator